app/src/main/java/com/duckduckgo/app/notification/NotificationRegistrar.kt

```diff
@@ -48,6 +48,8 @@ class NotificationRegistrar @Inject constructor(
     object NotificationId {
         const val ClearData = 100
         const val PrivacyProtection = 101
+        const val Article = 103
```

Comment:
Are you using 103 because 102 was the Search Notification? I'm pretty sure we can use 102, but if you don't want to risk it I'd a comment there to indicate why not

Reply:
I've added a comment just in case